android/titanium/src/java/org/appcelerator/titanium/util/TiMenuSupport.java

```diff
@@ -47,6 +51,16 @@ public boolean onCreateOptionsMenu(boolean created, Menu menu)
 		if (onCreate != null || onPrepare != null) {
 			created = true;
 		}
+
+		Drawable searchIconDrawable = null;
+		try {
+			searchIconDrawable = TiApplication.getInstance().getResources().getDrawable(
+				TiRHelper.getResource("drawable.abc_ic_search_api_material"), null);
+			searchIconDrawable.setColorFilter(Color.RED, PorterDuff.Mode.SRC_IN);
```

Comment:
I think you need to call `searchIconDrawable.mutate()` since you're receiving a global reference to the drawable here.